dvc/rwlock.py

```diff
@@ -91,8 +94,11 @@ def _check_no_readers(lock, info, path_infos):
             continue
 
         raise LockError(
-            "'{}' is busy, it is being read by:\n{}".format(
-                str(path_info), _infos_to_str(readers)
+            "'{path}' is busy, it is being read by:\n{process}"
```

Comment:
I think at this point it's better to this message compose logic to a function or string template. To avoid code duplication